client/js/postManager.js

```diff
@@ -0,0 +1,56 @@
+class PostManager {
+        
+    constructor(myUrl) {
+        this.url = myUrl;
+    }
+    
+    postData(inUrl, inData) {
+        
+        var outData = undefined;
+        
+        $.ajax({
+            url: this.url+"/"+inUrl,
+            type: "POST",
+            data: JSON.stringify(inData),
```

Comment:
`JSON.stringify()` is not necessary. jQuery will do this for you because you have specified `dataType: "json"`.